packages/flame/lib/src/events/flame_game_mixins/double_tap_dispatcher.dart

```diff
@@ -14,7 +14,7 @@ class DoubleTapDispatcherKey implements ComponentKey {
   int get hashCode => 20260645; // 'DoubleTapDispatcherKey' as hashCode
 
   @override
-  bool operator ==(dynamic other) =>
+  bool operator ==(Object other) =>
```

Comment:
I never seen it as a nillable object, so leaving this comment to find out as well 😬

Reply:
If it wouldn't be nullable it would be quite a big breaking change right? Since everyone doing `myObject == myNullableObject` would have to do null checks externally

Reply:
Having a nullable type here causes linter warnings as of 3.19.0-something